src/evox/algorithms/de_variants/ode.py

```diff
@@ -0,0 +1,138 @@
+from typing import Literal
+
+import torch
+
+from ...core import Algorithm, Mutable, Parameter, jit_class
+from ...utils import clamp
+
+
+@jit_class
+class ODE(Algorithm):
+    def __init__(
+        self,
+        pop_size: int,
+        lb: torch.Tensor,
+        ub: torch.Tensor,
+        base_vector: Literal["best", "rand"] = "rand",
+        num_difference_vectors: int = 1,
+        differential_weight: float | torch.Tensor = 0.5,
+        cross_probability: float = 0.9,
+        mean: torch.Tensor | None = None,
+        stdev: torch.Tensor | None = None,
+        device: torch.device | None = None,
+    ):
+        """
+        Initialize the DE algorithm with the given parameters.
+
+        Args:
```

Comment:
- The docstring should use Markdown and sphinx style notation for parameters.
- Since we already have type hints in the code, the docstring should exclude those type info.
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html
Like this:
```
:param pop_size: The size of the population.
:param lb: The lower bounds of the particle positions. Must be a 1D tensor.
```